extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -347,7 +348,8 @@ class AmpLightbox extends AMP.BaseElement {
    * @private
    */
   scrollHandler_() {
-    const currentScrollTop = this.element./*OK*/scrollTop;
+    // If scroll top is 0, it's set to 1 to avoid scroll-freeze issue.
+    const currentScrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
@muxin by setting `this.pos_` to 1, do you mean by calling `this.update_` ? or setting it directly in `scrollHandler` along with `currentScrollTop` ?